lib/bundler/cli/gem.rb

```diff
@@ -125,6 +125,15 @@ def run
         templates.merge!("CODE_OF_CONDUCT.md.tt" => "CODE_OF_CONDUCT.md")
       end
 
+      if ask_and_set(:rubocop, "Do you want to add rubocop as a dependency for gems you generate?",
+        "RuboCop is a static code analyzer that has out-of-the-box rules for many" \
+        "of the guidelines in the community style guide." \
+        "For more information, see the RuboCop docs (https://rubocop.readthedocs.io/en/latest/)" \
```

Comment:
FYI, the document URL of RuboCop is now below.
http://www.rubocop.org/en/latest/

Refer https://github.com/rubocop-hq/rubocop/blob/v0.58.2/rubocop.gemspec#L33